src/components/FlyModal/FlyModal.tsx

```diff
@@ -33,7 +33,7 @@ interface IProps extends IReactComponentProps {
 	closeTimeoutMS?: number;
 	hasIcon?: boolean;
 	isOpen?: boolean;
-	onRequestClose?: () => void;
+	onRequestClose?: () => boolean | void;
```

Comment:
Just curious, why is this boolean or void?

Reply:
Good catch. I initially made it so returning false on `onRequestClose` would stop the modal from closing but I'm going to ditch that for now.

Reply:
That makes sense. I wondered cause I've done the same thing when building modal components in the past. Just didn't see it being used and didn't know if there were plans to use it that way later on.